spec/requests/orders_spec.rb

```diff
@@ -0,0 +1,140 @@
+require "rails_helper"
+
+RSpec.describe OrdersController, type: :controller do
+  include ActiveJob::TestHelper
+
+  let(:user) { create :user }
+  let!(:product) { create(:product, status: :enabled) }
+
+  before { sign_in user }
+
+  describe "GET #index" do
+    let!(:order) { create :order, user: user, status: status }
+    
+    context "when get all orders success" do
+      let(:status) { :wait_confirm }
+
+      before { get :index }
+
+      it do
+        expect(response).to be_successful
+        expect(response).to render_template "index"
```

Comment:
```suggestion
        expect(response).to render_template :index
```

prefer dùng symbol nha em

Reply:
@nhuthh-2546 dạ okie anh để em update lại ạ